src/Body/EmojiGrid/style.css

```diff
@@ -0,0 +1,3 @@
+.grid-Emoji{
```

Comment:
```suggestion
.gridEmoji{
```
Be consistent with you naming style